index.html

```diff
@@ -22,8 +22,25 @@
   gtag('config', 'G-7960NHDJR9');
 </script>
 </head>
-
 <body>
+    <div class="blur">
+        <div class="mobile">
+            <ul class="menuMenu">
+                    <li class="nav-tag">
+                    <a class="portfolio"  href="#portfolio-sec">Portfolio</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#about-sec">About</a>
+                    </li>
+                    <li class="nav-tag">
+                    <a  href="#contact-sec">Contact</a>
+                    </li>
+            </ul>
+            <button type="button" class ="hamburger">
+                <i class="menuIcon fas fa-bars"> </i>
+                <i class="closeIcon fas fa-times"></i>
+            </button>
+        </div>
     <div class="container">
```

Comment:
- [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.